app/domain/authentication/authn_jwt/configuration_factory.rb

```diff
@@ -1,13 +1,17 @@
 module Authentication
   module AuthnJwt
-    # Factory that receives a vendor name and returns the appropriate JWT vendor configuration class
+    # Factory that receives an authenticator name and returns the appropriate JWT vendor configuration class
```

Comment:
wasn't this factory supposed to return each jwt vendor configuration (gitlab, jenkins etc)? and only jwt_authenticator is supposed to use this factory anyway

if so, why would we want the factory to receive authenticator name?

Reply:
Good question,

This authenticator is generic and will probably will handle multiple vendors that using JWT token.
In the future when we will want to implement a special logic (or to change the UX) to a specific vendorX
And Rest request will be /authn-vendor-x/..... and therefore the authenticator name will be authn-vendor-x